addons/redis/config/redis7-config.tpl

```diff
@@ -43,9 +43,10 @@ appendfsync everysec
 no-appendfsync-on-rewrite no
 auto-aof-rewrite-percentage 100
 auto-aof-rewrite-min-size 64mb
+aof-disable-auto-gc no
```

Comment:
What is the default value of this parameter? And what is its purpose? I couldn't find it in the official redis.conf file. This issue: https://github.com/redis/redis/issues/10561 mentions that it is just a testing parameter that exists in the internal code.

Reply:
the default value is no, I mentioned this parameter just to clarify that I used this parameter. I actually set the parameter to no in the backup target pod only while the users trigger the PITR.  As I see, using the testing parameter doesn't affect the existing cluster and the users who don't need PITR.